yarn_api_client/main.py

```diff
@@ -15,8 +15,7 @@ def get_parser():
     parser = argparse.ArgumentParser(
         description='Client for Hadoop® YARN API')
 
-    parser.add_argument('--host', help='API host')
-    parser.add_argument('--port', help='API port')
+    parser.add_argument('--endpoint', help='API endpoint (https://test.cluster.com:1234)')
```

Comment:
Could we use the default port in the help text?

Reply:
Changed to default RM https port.